modules/tensor_mechanics/include/utils/DeformationGradientTools.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
```

Comment:
Would it be better to call this file as "CohesiveZoneModelTools"?

Reply:
the methods defined here are generally applicable to any deformation gradient defined over a surface, the final decision is up to you

Reply:
I do not see any other applications that might need the deformation gradient over a surface. Let us call it "cohesiveZoneModelTools" for now. If we find more general usage, we can rename it later.

Reply:
done